requirements.txt

```diff
@@ -2,12 +2,26 @@ coverage==4.5.3
 coveralls==1.6.0
 pytest==4.3.1
 pytest-cov==2.6.1
+certifi==2019.3.9
+chardet==3.0.4
 Click==7.0
+coreapi==2.3.3
+coreschema==0.0.4
 Django==2.1.7
 django-cors-middleware==1.3.1
 django-extensions==2.1.6
+django-rest-swagger==2.2.0
 djangorestframework==3.9.2
+idna==2.8
+itypes==1.1.0
+Jinja2==2.10
+MarkupSafe==1.1.1
+openapi-codec==1.3.2
 pip-tools==3.5.0
 PyJWT==1.7.1
 pytz==2018.9
+requests==2.21.0
+simplejson==3.16.0
```

Comment:
Nice work .
Could you remove dependencies that are not necessary .
Only have those you sure you installed